### PR TITLE
extend battery_monitor with additional rosparams

### PR DESCRIPTION
**Pull request by benmaidel**
Added rosparams for:
- color_charging
- color_warning
- color_error
- color_critical
- interval_duration_warning
- interval_duration_error
- interval_duration_critical

The default values for the rosparams are the same as before. So if the new parameters are not on the parameter server loaded the behavior of this node should be the same as before.

### CODE REVIEW

**Comment by benmaidel** [comment]
tested on mrl-5 :heavy_check_mark:

File: cob_monitoring/src/battery_monitor.py [hunk 1]
```diff
@@ -45,6 +45,39 @@ def __init__(self):
...
+        if rospy.has_param("~color_critical"):
+            color_critical_param = rospy.get_param("~color_critical")
+        else:
+            # set to cyan as default
+            color_critical_param = [1.0, 0.0, 0.0, 1.0]
+        self.color_critical = ColorRGBA(*color_critical_param)
+
+        self.interval_duration_warning = rospy.get_param("~interval_duration_warning", 30.0)
+        self.interval_duration_error = rospy.get_param("~interval_duration_error", 30.0)
+        self.interval_duration_critical = rospy.get_param("~interval_duration_critical", 30.0)
```
Comment (fmessmer) [inline cob_monitoring/src/battery_monitor.py]: this was previously set to `5.0`

File: cob_monitoring/src/battery_monitor.py [hunk 2]
```diff
@@ -45,6 +45,39 @@ def __init__(self):
...
+
+        if rospy.has_param("~color_critical"):
+            color_critical_param = rospy.get_param("~color_critical")
+        else:
+            # set to cyan as default
+            color_critical_param = [1.0, 0.0, 0.0, 1.0]
+        self.color_critical = ColorRGBA(*color_critical_param)
+
+        self.interval_duration_warning = rospy.get_param("~interval_duration_warning", 30.0)
+        self.interval_duration_error = rospy.get_param("~interval_duration_error", 30.0)
```
Comment (fmessmer) [inline cob_monitoring/src/battery_monitor.py]: this was previously set to `15.0`

**Review by fmessmer — CHANGES_REQUESTED** [review]

File: cob_monitoring/src/battery_monitor.py (hunk 1 above)
Comment (benmaidel) [reply cob_monitoring/src/battery_monitor.py]: copy past, nice catch

File: cob_monitoring/src/battery_monitor.py (hunk 2 above)
Comment (benmaidel) [reply cob_monitoring/src/battery_monitor.py]: copy past, nice catch

**Review by fmessmer — APPROVED** [review]

**Pull request merged**